### PR TITLE
[pack] Zip up the crossgen symbols so they appear as build artifacts

### PR DESCRIPTION
**Pull request by paulbatum**

### CODE REVIEW

**Comment by fabiocav** [comment]
Kicked off a build with packaging enabled. Looks good!

File: build.ps1 [hunk 1]
```diff
@@ -205,6 +205,12 @@ function CreateZips([string] $runtimeSuffix) {
         Remove-Item $tempXdtDir -Recurse
     }
 
+    # Zip up symbols for builds with runtime embedded
+    if ($runtimeSuffix -eq  "") {
+        ZipContent "$buildOutput\publish.win-x86\Symbols" "$buildOutput\Functions.Symbols.$extensionVersion-alpha.win-x86.zip"
+        ZipContent "$buildOutput\publish.win-x64\Symbols" "$buildOutput\Functions.Symbols.$extensionVersion-alpha.win-x64.zip"
```
[the remark this answers is not in the document]
Comment (fabiocav) [reply build.ps1]: Verified that the script works (a local run produces the expected output), but DiaSymReader x86 is likely missing/not in the path.

File: build.ps1 [hunk 2]
```diff
@@ -43,7 +43,7 @@ function CrossGen([string] $runtime, [string] $publishTarget, [string] $privateS
     dotnet publish .\src\WebJobs.Script.WebHost\WebJobs.Script.WebHost.csproj -r $runtime -o "$selfContained" -v q /p:BuildNumber=$buildNumber    
 
     # Modify web.config for inproc
-    dotnet tool install -g dotnet-xdt --version 2.1.0-rc.1
+    dotnet tool install -g dotnet-xdt --version 2.1.0 2> $null
```
Comment (fabiocav) [inline build.ps1]: We could move this elsewhere, but for now, I'm just sending stderr nowhere. Also updated to 2.1.0

**Comment by fabiocav** [comment]
Looks like adding the path resolved the x86 PDB generation issue. We should be good to go with this

File: build.ps1 [hunk 3]
```diff
@@ -205,6 +220,12 @@ function CreateZips([string] $runtimeSuffix) {
         Remove-Item $tempXdtDir -Recurse
     }
 
+    # Zip up symbols for builds with runtime embedded
+    if ($runtimeSuffix -eq  "") {
+        ZipContent "$buildOutput\publish.win-x86\Symbols" "$buildOutput\Functions.Symbols.$extensionVersion-alpha.win-x86.zip"
+        ZipContent "$buildOutput\publish.win-x64\Symbols" "$buildOutput\Functions.Symbols.$extensionVersion-alpha.win-x64.zip"
```
Comment (safihamid) [inline build.ps1]: If this continues to be a problem maybe we should pass the /DiasymreaderPath to crossgen.exe and see if it works? last time I tried, it had issues but might have been fixed by now.

 Debugging Parameters
    /CreatePDB <Dir to store PDB> [/lines [<search path for managed PDB>] ]
        When specifying /CreatePDB, the native image should be created
        first, and <assembly name> should be the path to the NI.
    /DiasymreaderPath <Path to diasymreader.dll>
        - Specifies the absolute file path to diasymreader.dll to be used.

**Pull request merged**